core/string/ustring.cpp

```diff
@@ -3166,6 +3166,46 @@ int String::findn(const String &p_str, int p_from) const {
 	return -1;
 }
 
+int String::findn(const char *p_str, int p_from) const {
+	if (p_from < 0) {
+		return -1;
+	}
+
+	int src_len = strlen(p_str);
```

Comment:
One concern is the universality of this, and uniformity, as `find` doesn't use it